frontend/packages/topology/src/components/page/TopologyPageToolbar.tsx

```diff
@@ -33,7 +38,7 @@ const TopologyPageToolbar: React.FC<TopologyPageToolbarProps> = observer(
         {showGraphView && !isMobile ? (
           <Popover
             aria-label={t('topology~Shortcuts')}
-            bodyContent={getTopologyShortcuts(t)}
+            bodyContent={getTopologyShortcuts(t, { extensions })}
```

Comment:
`extensions` is generic, should rename it to `supportedFileType` or something else